app/src/test/java/com/microsoft/device/samples/dualscreenexperience/domain/order/usecases/GetAllOrdersUseCaseTest.kt

```diff
@@ -0,0 +1,65 @@
+/*
+ *
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ *
+ */
+
+package com.microsoft.device.samples.dualscreenexperience.domain.order.usecases
+
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule
+import com.microsoft.device.samples.dualscreenexperience.domain.order.testutil.MockOrderDataSource
+import com.microsoft.device.samples.dualscreenexperience.domain.order.testutil.firstOrderEntity
+import com.microsoft.device.samples.dualscreenexperience.domain.order.testutil.firstSubmittedOrder
+import com.microsoft.device.samples.dualscreenexperience.domain.order.testutil.firstSubmittedOrderEntity
+import com.microsoft.device.samples.dualscreenexperience.domain.order.testutil.getOrAwaitValue
+import kotlinx.coroutines.runBlocking
+import org.hamcrest.MatcherAssert
+import org.hamcrest.core.Is
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+
+class GetAllOrdersUseCaseTest {
+
+    private lateinit var getAllOrdersUseCase: GetAllSubmittedOrdersUseCase
+    private lateinit var mockRepo: MockOrderDataSource
+
+    @get:Rule
+    var instantTaskExecutorRule = InstantTaskExecutorRule()
+
+    @Before
+    fun setup() {
+        mockRepo = MockOrderDataSource()
+        getAllOrdersUseCase = GetAllSubmittedOrdersUseCase(mockRepo)
+    }
+
+    @Test
+    fun getEmptyListWhenNoOrders() = runBlocking {
+        val resultValue = getAllOrdersUseCase.get().getOrAwaitValue()
+
+        MatcherAssert.assertThat(resultValue, Is.`is`(emptyList()))
```

Comment:
To make this easier to write, you can add this import alias `import org.hamcrest.core.Is.'is' as iz` and then use it like `MatcherAssert.assertThat(resultValue, iz(emptyList()))`